dive_core/command_hierarchy.cpp

```diff
@@ -1801,13 +1803,28 @@ void CommandHierarchyCreator::AppendRegNodes(const IMemoryManager &mem_manager,
         if (reg_info_ptr == nullptr)
             reg_info_ptr = &temp;
 
-        DIVE_ASSERT(!reg_info_ptr->m_is_64_bit);
+        uint64_t reg_value = reg_pair.m_reg_value;
+        if (reg_info_ptr->m_is_64_bit)
+        {
+            RegPair  new_reg_pair;
+            uint64_t new_pair_addr = va_addr + dword * sizeof(uint32_t);
+            DIVE_VERIFY(mem_manager.CopyMemory(&new_reg_pair,
+                                               submit_index,
+                                               new_pair_addr,
+                                               sizeof(new_reg_pair)));
+
+            // Sometimes the upper 64-bits are not set
```

Comment:
should this comment be upper 32-bits of the 64bit register?

Reply:
Good catch. I meant to say "upper part of 64-bits", but changing it 32-bits makes more sense